stdlib/asm/collections/mmr.masm

```diff
@@ -175,3 +196,43 @@ export.unpack
   # assert on the resulting hash (11 cycles)
   assert_eqw
 end
+
+#! Computes the hash of the given Mmr.
```

Comment:
nit: I'd say `MMR` instead of `Mmr`.

Also, We should probably add a sentence in here to say that the advice provider is updated with MMR peaks.